sql/derived_tables/finance_transaction_purchase_order.sql

```diff
@@ -10,6 +10,9 @@ SELECT
     ft.amount AS transaction_amount,
     ft.currency AS transaction_currency,
     ft.data #>> '{expenseClassId}' AS transaction_expense_class_id,
+    ec.data #>> '{code}'AS transaction_expense_class_code,
+    ec.data #>> '{name}'AS transaction_expense_class_name,
```

Comment:
Although not strictly necessary, it would be good I think to add a space before `AS`.